impl/maven-jline/src/main/java/org/apache/maven/jline/MessageUtils.java

```diff
@@ -49,8 +49,11 @@ public static void systemInstall() {
     public static void systemInstall(Consumer<TerminalBuilder> builderConsumer, Consumer<Terminal> terminalConsumer) {
         MessageUtils.terminal = new FastTerminal(
                 () -> {
-                    TerminalBuilder builder =
-                            TerminalBuilder.builder().name("Maven").dumb(true);
+                    TerminalBuilder builder = TerminalBuilder.builder()
+                            .jansi(false)
+                            .jna(false)
+                            .jni(true)
+                            .name("Maven");
```

Comment:
We may still want to have `dumb(true)`.
For example if we are on an unsupported os/architecture combination, we'll end up with a dumb terminal, but with an annoying warning (and nothing the user can do)...